Automattic/jetpack
# Release config package

Fixes #20941

#### Changes proposed in this Pull Request:

* Releases the Config package, which we updated to include the Post List package

#### Jetpack product discussion



N/A

#### Does this pull request change what data or activity we track or use?



N/A

#### Testing instructions:






* Enable the Post List package via the Config package. See https://github.com/Automattic/jetpack/pull/20938
